react/reducer_tests.rst

```diff
@@ -0,0 +1,84 @@
+.. _reducer_tests-label:
+
+=============================
+Write Tests For Your Reducers
+=============================
+
+Reducer Tests
+=============
+
+Since reducers are just pure functions with input and output they are really
+easy to write unit tests for. We'll start by adding a test for the
+:file:`ADD_TODO` action in a file called :file:`reducers/faq.test.js`:
```

Comment:
nothing is really easy, maybe it is easy for you, but that does not mean it is easy for the whole world :)

Please remove or change "really easy"  and one sentence per line :)